deployment-script/ConfiguredScript.sol

```diff
@@ -0,0 +1,72 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity ^0.8.0;
+
+import "../lib/forge-std/src/Script.sol";
+import "../lib/forge-std/src/console2.sol";
```

Comment:
```suggestion
import "../lib/forge-std/src/console.sol";
```
console2 is the same as console now (see [here](https://morpholabs.slack.com/archives/C02P3QE2Q8G/p1726145495473169))